content/en/continuous_integration/pipelines/gitlab.md

```diff
@@ -272,3 +273,6 @@ Log files larger than 1GiB are truncated.
 [14]: /continuous_integration/pipelines/gitlab/?tab=gitlabcom#correlate-infrastructure-metrics-to-jobs
 [15]: /continuous_integration/pipelines/gitlab/?tab=gitlabcom#view-error-messages-for-pipeline-failures
 [16]: /account_management/teams/
+[17]: https://app.datadoghq.com/ci/setup/pipeline?provider=gitlab
+[18]: https://app.datadoghq.com/ci/pipeline-executions?query=ci_level%3Apipeline%20%40ci.provider.name%3Agitlab&index=cipipeline
```

Comment:
This won't get the user to anything where they can see logs or pipelines that have logs exclusively. I don't think we have a deep link for this

Reply:
```suggestion
```